spec/cspec/c/gen-config-thy.py

```diff
@@ -170,6 +170,7 @@
     'CONFIG_CTZ_64': (bool, None),
     'CONFIG_CLZ_NO_BUILTIN': (bool, None),
     'CONFIG_CTZ_NO_BUILTIN': (bool, None),
+    'CONFIG_MAX_NUM_IOAPIC': (nat, 'maxNumIOAPIC'),
```

Comment:
this is interesting; does this mean that if we need this for any specific architecture, we create a Kernel_Config entry for all architectures? i.e. does ARM now have a `maxNumIOAPIC`?

Reply:
yes, the tool has no way of distinguishing an option that is unset for a specific architecture from an option that is unset because it is not part of that architecture. That means we get all names for all architectures.